tests/test.bats

```diff
@@ -499,3 +499,27 @@ _healthcheck_wait ()
 	### Cleanup ###
 	make clean
 }
+
+@test "PHPCS Coding standards check" {
+	[[ $SKIP == 1 ]] && skip
+
+	### Setup ###
+	make start
+	_healthcheck_wait
+
+	### Tests ###
+
+	# Check PHPCS libraries loaded
+	run docker exec -u docker "$NAME" bash -lc 'phpcs -i'
```

Comment:
Output matching here is ambiguous, since `[[ "${output}" =~ "Drupal" ]]` will match `*Drupal*`.

How about this:

```
[[ "${output}" =~ "Drupal, DrupalPractice" ]]
[[ "${output}" =~ "WordPress-Extra, WordPress-Docs, WordPress-VIP, WordPress-Core and WordPress" ]]
```

Reply:
The only thing I would fear is tha tthey don't load in the same order every time for some reason so I'd rather change to ` Drupal, ` and ` DrupalPractice, ` where there is a space before and there is a `, ` after.

Reply:
I wouldn't expect the order to change sporadically without changes to dependency versions/etc. The two explicit lines just look cleaner to me. In a rare situation when the order does change, updating the test won't be a big deal.

Reply:
Well... looks like it does print then in a sporadic order:

```
The installed coding standards are Squiz, PSR12, PEAR, PSR1, MySource, Zend, PSR2, Drupal, DrupalPractice, WordPress-Extra, WordPress-Docs, WordPress-VIP, WordPress-Core and WordPress
```

```
The installed coding standards are PSR12, PEAR, Zend, PSR1, MySource, PSR2, Squiz, Drupal, DrupalPractice, WordPress-Extra, WordPress-Docs, WordPress, WordPress-VIP and WordPress-Core
```